ginga/examples/configs/channel_Image.cfg

```diff
@@ -0,0 +1,118 @@
+#
+# Image channel preferences file
+#
+# Place this in file under ~/.ginga with the name "channel_Image.cfg" for
+# channel name "Image". This file WILL BE OVERWRITTEN by the Preferences
+# plugin (see below). NOT ALL PARAMETERS IN GUI CAN BE ADJUSTED HERE.
+#
+# Alternately, the settings can be adjusted via Preferences local plugin
+# within Ginga. NOT ALL PARAMETERS CAN BE ADJUSTED VIA GUI but users can
+# normally ignore the hidden ones.
+#
+
+# ---------------
+# Color Distribution
+
+color_algorithm = 'linear'
+
+# ---------------
+# Color Mapping
+
+color_map = 'gray'
+color_hashsize = 65535
```

Comment:
As the hash size grows smaller, and depending on the values in the image, the cut levels and the color distribution algorithm (linear, log, etc), distinct image values will be hashed into the same buckets, so subtle gradations could show up as posterization in the image, especially with the standard rgbmapper mapping to 24 bpp displays.  

Practically speaking, with the default value of the hash size as high as it is and the limitation of current displays there is no need for the user to set the value.  That is why I removed it from the `Preferences` plugin.